packages/reporting-server/rest_server/__main__.py

```diff
@@ -1,18 +1,48 @@
-import uvicorn
+import asyncio
 
-from .app import app
-from .app_config import app_config
+from uvicorn import Config, Server
 
+from .app import get_app
+from .app_config import SystemMode, app_config
 
-def main():
-    uvicorn.run(
-        app,
+
+class MyServer(Server):
+    # pylint: disable=invalid-overridden-method
```

Comment:
Can you add a comment explaining why this is ok/needed?

Reply:
Sure thing! Thanks!